packages/plugins/src/plugins/morpho/lendingFetcher.ts

```diff
@@ -0,0 +1,129 @@
+import { EvmNetworkIdType } from '@sonarwatch/portfolio-core';
+import { getAddress } from 'viem';
+import { Fetcher, FetcherExecutor } from '../../Fetcher';
+import {
+  morphoMarketsCachePrefix,
+  networkIdToMorphoContract,
+  platformId,
+} from './constants';
+import { getEvmClient } from '../../utils/clients';
+import { ElementRegistry } from '../../utils/elementbuilder/ElementRegistry';
+import { Cache } from '../../Cache';
+import { MorphoMarketRes } from './types';
+import { morphoContractABI } from './utils/abis';
+
+export function getLendingFetcher(networkId: EvmNetworkIdType): Fetcher {
+  const executor: FetcherExecutor = async (owner: string, cache: Cache) => {
+    const markets = await cache.getItem<MorphoMarketRes['markets']['items']>(
+      morphoMarketsCachePrefix,
+      {
+        networkId,
+        prefix: morphoMarketsCachePrefix,
+      }
+    );
+    if (!markets || markets.length === 0) {
+      return [];
+    }
+
+    const client = getEvmClient(networkId);
+    const multicallResponse = await client.multicall({
+      contracts: markets.flatMap((market) => [
+        {
+          address: getAddress(networkIdToMorphoContract[networkId]),
+          functionName: 'position',
+          abi: morphoContractABI,
+          args: [market.uniqueKey, owner],
+        },
+        {
+          address: getAddress(networkIdToMorphoContract[networkId]),
+          functionName: 'market',
+          abi: morphoContractABI,
+          args: [market.uniqueKey],
+        },
+      ]),
+    });
+
+    const elementRegistry = new ElementRegistry(networkId, platformId);
+    for (let i = 0; i < multicallResponse.length; i += 2) {
+      const partialMarket = markets[i];
+      const userPosition = multicallResponse[i];
+      const marketData = multicallResponse[i + 1];
+
+      if (userPosition.error || marketData.error) {
+        continue;
+      }
+
+      const [supplyShares, borrowShares, collateral] = userPosition.result as [
+        bigint,
+        bigint,
+        bigint
+      ];
+
+      if (
+        supplyShares === BigInt(0) &&
+        borrowShares === BigInt(0) &&
+        collateral === BigInt(0)
```

Comment:
```suggestion
        supplyShares === zeroBigInt &&
        borrowShares === zeroBigInt &&
        collateral === zeroBigInt
```

Ref: https://octavfi.slack.com/archives/C08KX01HLMS/p1744043486173509

Reply:
Please use this `zeroBingInt` as much as possible below